File: mi_prime_paquete/src/suscriptor.cpp
#include "ros/ros.h"
#include "turtlesim/Pose.h"


turtlesim::Pose pose_turtle;

void poseCallback(const turtlesim::Pose::ConstPtr& msg){

    pose_turtle.x = msg->x;
    pose_turtle.y = msg->y;
    
    ROS_INFO("Se la posicion de  la tortuga en x = [%.3f] y = [%.3f]",pose_turtle.x,pose_turtle.y);
}


int main(int argc, char **argv)
{
  ros::init(argc, argv, "nodo_suscriptor");

  ROS_INFO("Se inicializo el nodo suscriptor");
  ros::NodeHandle n;
  ros::Subscriber pose_sub = n.subscribe("/turtle1/pose",10, poseCallback);

  //ros::spin();
  ROS_INFO("Despues del spin");
  return 0;
}

File: mi_primer_paquete/scripts/suscriptor.py
#!/usr/bin/env python3
import rospy
from turtlesim.msg import Pose
from geometry_msgs.msg import Twist


def poseCallback(data):
    rospy.loginfo("La posicion de la tortuga")  

def suscriptor():
    rospy.init_node('suscriptor',anonymous=True)

    sub = rospy.Subscriber('/turtle1/pose', Pose, poseCallback)
    rospy.spin()

if __name__ == '__main__':
        try:
             suscriptor()
        except rospy.ROSInterruptException:
             pass

